Debugging/NodeJs.md

```diff
@@ -0,0 +1,29 @@
+
+# NodeJs
+
+Installation instructions of the Tessel CLI can be found here: http://tessel.github.io/t2-start/
```

Comment:
Can we make the url into an official markdown link?

`[http://tessel.github.io/t2-start/](http://tessel.github.io/t2-start/)`

Also, I believe the text should be updated to read:

Installation instructions **for** the Tessel CLI can be found here:

I highlighted the changed word in bold.